presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpLocationFactory.java

```diff
@@ -35,9 +36,9 @@
     private final URI baseUri;
 
     @Inject
-    public HttpLocationFactory(InternalNodeManager nodeManager, HttpServerInfo httpServerInfo)
+    public HttpLocationFactory(InternalNodeManager nodeManager, HttpServerInfo httpServerInfo, InternalHttpClientConfiguration config)
```

Comment:
Same here, but if renaming is going to churn everything, I'm OK with sticking with the existing convention of ignoring this distinction ;-)